spec/group_spec.rb

```diff
@@ -1,50 +1,17 @@
 require_relative 'spec_helper'
 
 describe AnsibleTowerClient::Group do
-  let(:groups_body) do
-    {:count => 2, :next => nil,
-     :previous => nil,
-     :results => [{:id => 1, :type => 'group',
-                   :url => '/api/v1/groups/1/', :name => 'test1'},
-                  {:id => 2, :type => 'groups',
-                   :url => '/api/v1/groups/2/', :name => 'test2'}]}.to_json
-  end
-
-  let(:one_result) do
-    {:id => 1, :url => '/api/v1/groups/1/', :name => 'test1'}.to_json
-  end
-
   let(:api_connection) { instance_double("Faraday::Connection", :get => get) }
```

Comment:
Do we need the get method as part of this instance_double? (on any of the specs that don't actually call ':get')